includes/migration-guide/retargeting/ef/entity-framework-version-must-match-net.md

```diff
@@ -2,7 +2,7 @@
 
 |   |   |
 |---|---|
-|Details|The entity framework version should be matched with the .NET framework version. Entity Framework 5 is recommended for .NET Framework 4.5. There are some known issues with EF 4.x in a .NET Framework 4.5 project around <xref:System.ComponentModel.DataAnnotations>. In .NET 4.5, these were moved to a different assembly, so there are issues determining which annotations to use.|
+|Details|The entity framework version should be matched with the .NET Framework version. Entity Framework 5 is recommended for .NET Framework 4.5. There are some known issues with EF 4.x in a .NET Framework 4.5 project around <xref:System.ComponentModel.DataAnnotations>. In .NET 4.5, these were moved to a different assembly, so there are issues determining which annotations to use.|
```

Comment:
I think this should be .NET Framework not just .NET.
I won't apply it before someone confirms.

```suggestion
|Details|The entity framework version should be matched with the .NET Framework version. Entity Framework 5 is recommended for .NET Framework 4.5. There are some known issues with EF 4.x in a .NET Framework 4.5 project around <xref:System.ComponentModel.DataAnnotations>. In .NET 4.5 Framework, these were moved to a different assembly, so there are issues determining which annotations to use.|
```

Reply:
@bricelam @smitpatel can you help here?

Also, Entity Framework should have capital letters on "The Entity Framework version..."

Reply:
Let's go with the following:

```suggestion
|Details|The Entity Framework (EF) version should be matched with the .NET Framework version. Entity Framework 5 is recommended for .NET Framework 4.5. There are some known issues with EF 4.x in a .NET Framework 4.5 project around <xref:System.ComponentModel.DataAnnotations>. In .NET Framework 4.5, these were moved to a different assembly, so there are issues determining which annotations to use.|
```

Reply:
@mairaw, Should I commit this suggestion ?